update local agent documentation (#392)
grafana/agent: production/README.md

> ## Running the Agent locally

Reviewer: ```suggestion
## Running the Agent locally

### Linux (Debian or RedHat-based distributions)
```

> The deb and rpm packages provides needed script to run the grafana-agent as a long-living process. 

Reviewer: ```suggestion
The deb and rpm packages provides systemd scripts to run the Agent as a long-living process. 
```

> Don't forget to run `systemctl status grafana-agent`to check service status.

Reviewer: ```suggestion
You can run `systemctl status grafana-agent` to check service status:
```

```
systemctl status grafana-agent
● grafana-agent.service - Monitoring system and forwarder
     Loaded: loaded (/lib/systemd/system/grafana-agent.service; disabled; vendor preset: enabled)
     Active: inactive (dead)
       Docs: https://github.com/grafana/agent/blob/master/docs/README.md
```

> ## Use the example Kubernetes configs

Reviewer: ```suggestion
### Other Systems 
Binary builds for other systems are provided, and you must provide your own configuration to configure the Agent to run as a background process. Support for system packages for more platforms will be provided over time.

## Use the example Kubernetes configs
```
